src/Tools/IdeCoreBenchmarks/SegmentedListBenchmarks_InsertRange.cs

```diff
@@ -40,6 +41,17 @@ public void GlobalSetup()
             _segmentedInsertValuesObject = new SegmentedArray<object?>(100);
         }
 
+        [Benchmark(Description = "AddToSegmentedList<object>", Baseline = true)]
+        public void AddList()
```

Comment:
This should move to `SegmentedListBenchmarks_Add`

Reply:
I was going to undo the benchmarks and not merge them, unless you think I should go ahead and move this test and include it if this merges.

Reply:
➡️ Now resolved